.travis.yml

```diff
@@ -1,7 +1,15 @@
 language: java
 
 jdk:
-- oraclejdk8
+- oraclejdk11
+
+addons:
+  apt:
+    sources:
+      - mongodb-upstart
+    packages:
+      - mongodb-org-server
+      - mongodb-org-shell
```

Comment:
from what we saw yesterday, dont you think we dont need them anymore as the tests only work with mongo instance started from gradle?